src/frontend/src/views/Data/components/SamplesView/components/SampleTableModalManager/components/UsherTreeFlow/index.tsx

```diff
@@ -14,14 +15,20 @@ interface Props {
   shouldStartUsherFlow: boolean;
 }
 
+const USHER_DBS: Record<Pathogen, string> = {
```

Comment:
This feels like something the FE shouldn't need to know. That said, it's what we were doing in the past, the FE knew the DB_PARAM, this is just generalizing it, so it's not any worse than it was previously. But if you can see a clean/easy way to make this a piece of info the BE is returning with the rest of the USHER options info, that would be a nice-to-have.

Reply:
Can this type should use the `PathogenConfigType`?  This was originally just for the strings.ts files, but I've been using it for anything that is pathogen-specific. `src/components/DownloadMetadataTemplate/prepMetadataTemplate.ts` has and example on line 19.

Reply:
Plus one to this doesn't really belong on the FE, but also plus one to doing something practical